identity-service/src/routes/notifications.js

```diff
@@ -377,11 +435,12 @@ module.exports = function (app) {
         ],
         include: [{
           model: models.SolanaNotificationAction,
-          required: true,
+          required: false,
```

Comment:
@jowlee is this safe? Without this change, we don't receive these new notifs since we don't have corresponding actions

Reply:
This should be safe, I'd just double-check that there's no unchecked references to the action